canonical/ubuntu.com
# Update "no-support" text on Ubuntu pro pricing page to "self-support"

## Done

Changed the support text on the Ubuntu pricing page (https://ubuntu.com/pricing/pro) from "no support" to "self-support" as suggested.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/pro
- Check that in the tables `No Support` is now `Self Support`


## Issue / Card

Fixes #12465
